docs/root/intro/version_history.rst

```diff
@@ -44,7 +44,7 @@ Features:
 - android: create simple persistent SharedPreferencesStore (:issue: `#2319 <2319>`)
 - iOS: A documentation archive is now included in the GitHub release artifact (:issue: `#2335 <2335>`)
 - api: improved C++ APIs compatibility with Java / Kotlin / Swift (:issue `#2362 <2362>`)
-- api: add option to use the a ``getaddrinfo``-based system DNS resolver instead of c-ares (:issue: `#2419 <2419>`)
+- Android: default to use a ``getaddrinfo``-based system DNS resolver instead of c-ares (:issue: `#2419 <2419>`)
```

Comment:
for whatever reason Android is not capitalized in other occurences in this file. Up to you whether you want to stay consistent or not.

Reply:
Seems like we should honour brand name capitalization. As someone whose name is frequently mis-capitalized this might be a touchy issue for me 😅.